### DISCUSSION TITLE
How to get free learning resource on Github for data science?

### DISCUSSION THREAD

**Discussion by ezitwelve** [community/community#20861] [Discussions] [answered]
Any thing would be helpful

**Comment by AnuraghSarkar** [comment, marked as answer]
I personally like this https://github.com/ossu/data-science

**Comment by thaisonzdx** [comment]
I think its can help you:

- [Data Science for Beginners](https://github.com/microsoft/Data-Science-For-Beginners)
- [Machine Learning for Beginners](https://github.com/microsoft/ML-For-Beginners)
- [Python Data Science Handbook](https://github.com/jakevdp/PythonDataScienceHandbook)

**Comment by xaviedoanhduy** [comment]
hello, I am not sure, but it seems that if you are a student at university like me, it seems that github will support us with a package for students, see: https://github.com/education/students

**Reply by adambht** [reply]
Yes, that’s correct! GitHub offers the Student Developer Pack, which provides free access to a variety of tools and resources for students. You’ll need to verify your student status (like uploading proof of enrollment) to get access. It’s definitely worth checking out!

**Comment by francescousseau** [comment]
💡 **Free Learning Resources on GitHub for Data Science**

GitHub is an open laboratory for anyone who wants to learn Data Science in a practical way. Below are some repositories and curated collections that are extremely useful for students, professionals, and enthusiasts:

[Microsoft/Data-Science-For-Beginners](https://github.com/microsoft/Data-Science-For-Beginners?utm_source=chatgpt.com)
 → Free 10-week / 20-lesson curriculum with interactive quizzes.

[jakevdp/PythonDataScienceHandbook](https://github.com/jakevdp/PythonDataScienceHandbook?utm_source=chatgpt.com)
 → Notebooks from the classic Python Data Science Handbook.

[academic/awesome-datascience](https://github.com/academic/awesome-datascience?utm_source=chatgpt.com)
 → A wide list of courses, books, tools, and tutorials.

[krzjoa/awesome-python-data-science](https://github.com/krzjoa/awesome-python-data-science?utm_source=chatgpt.com)
 → Curated Python libraries for Data Science.

[ossu/data-science](https://github.com/ossu/data-science?utm_source=chatgpt.com)
 → Complete self-taught curriculum with links to free courses.

[alastairrushworth/free-data-science](https://github.com/alastairrushworth/free-data-science?utm_source=chatgpt.com)
 → A collection of free Data Science resources.

[awesomedata/awesome-public-datasets](https://github.com/awesomedata/awesome-public-datasets?utm_source=chatgpt.com)
 → Large collection of public datasets for practice and projects.

[practical-tutorials/project-based-learning](https://github.com/practical-tutorials/project-based-learning?utm_source=chatgpt.com)
 → Tutorials and examples based on real-world projects.

[youssefHosni/Awesome-AI-Data-Guided-Projects](https://github.com/youssefHosni/Awesome-AI-Data-Guided-Projects?utm_source=chatgpt.com)
 → Project ideas guided by AI and Data Science, great for building a portfolio.

[ElizaLo/Data-Science](https://github.com/ElizaLo/Data-Science?utm_source=chatgpt.com)
 → Collection of tasks, books, articles, and useful tools for Data Science.

✨ These resources are excellent starting points to build projects, strengthen your portfolio, and deepen your understanding of modern Data Science practices.
